src/components/autocomplete/js/highlightDirective.js

```diff
@@ -31,13 +31,17 @@ angular
  * </hljs>
  */
 
-function MdHighlight () {
+function MdHighlight ($interpolate, $parse) {
   return {
     terminal: true,
-    scope: false,
     controller: 'MdHighlightCtrl',
-    link: function (scope, element, attr, ctrl) {
-      ctrl.init(attr.mdHighlightText);
+    compile: function mdHighlightCompile(tElement, tAttr) {
```

Comment:
Performance. The compile function is called only once by the repeater (link is called for each cloned item). This makes less of a difference now that we're not using ng-repeat, but it's still there.